### PR TITLE
Top-level merge (known top-level multi-sub)

### PR DESCRIPTION
**Pull request by jacogr**
This one has only has `<scam>.netilify.com` usages - even if you try and find info on the top-level (now unavailable), it refers to "wallet selection" for the top-level domain.

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -6574,7 +6574,7 @@
 		"pancakeswap-finance-claim.site",
 		"pancakeswap-finance-liquidity-pool.site",
 		"pancakeswap-finance-swap-liquidity-v2.site",
-		"pancakeswap-finance.cake-token.in",
+		"cake-token.in",
```
Comment (jacogr) [inline all.json:6577]: Google also lists this top-level as a scam.

File: all.json [hunk 2]
```diff
@@ -6619,7 +6619,6 @@
 		"pancakeswap.finance.alifdashproject.com",
 		"pancakeswap.finance.biswap2.com",
 		"pancakeswap.finance.bitminer-cloud.com",
-		"pancakeswap.finance.cake-token.in",
```
Comment (jacogr) [inline all.json:6622]: Listed above via `cake-token.in`, removed here.

File: all.json [hunk 3]
```diff
@@ -6619,17 +6619,12 @@
...
-		"pancakeswap.finance.fastservicetrack.com",
+		"fastservicetrack.com",
```
Comment (jacogr) [inline all.json:6622]: Also listed by Google

File: all.json [hunk 4]
```diff
@@ -6619,25 +6619,16 @@
 		"pancakeswap.finance.alifdashproject.com",
 		"pancakeswap.finance.biswap2.com",
 		"pancakeswap.finance.bitminer-cloud.com",
-		"pancakeswap.finance.cake-token.in",
-		"pancakeswap.finance.fastservicetrack.com",
-		"pancakeswap.finance.goodsservicedogs.com",
-		"pancakeswap.finance.goswap.fastservicetrack.com",
+		"fastservicetrack.com",
+		"goodsservicedogs.com",
```
Comment (jacogr) [inline all.json:6623]: Multiple hist - site now defunct. Only mention on google does point back to this list, nowhere else.

**Review by ical10 — APPROVED** [review]

**Pull request merged**